### ISSUE TITLE
Not Highlighting Variables in {{}}

### ISSUE THREAD

**Issue by BeardedSwitchDragon** [benjohnson/atom-django-templates#40]
django-templates isn't highlighting my template variables enclosed in {{}}
![screen shot 2019-02-25 at 6 56 25 pm](https://user-images.githubusercontent.com/26681093/53322311-23e58000-392f-11e9-9686-56c9e7dc63a6.png)

**Comment by esmnt** [comment]
This is true for every django package i tried so far. Double brackets, variable name and filters are all the same color. I'm trying to figure out how to highlight them properly, like this, but I'm new to Atom.
<img width="302" alt="Django Variable Example" src="https://user-images.githubusercontent.com/62053500/76399432-a0f36400-638f-11ea-9b8e-32521dc69802.png">
